src/artifact_providers/github.ts

```diff
@@ -81,14 +64,14 @@ export class GithubArtifactProvider extends BaseArtifactProvider {
     const per_page = 100;
 
     // https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#artifacts
```

Comment:
Since we're not using this format anymore:
```suggestion

```

Reply:
I mean it is still the same endpoint 😀 I think it is good to keep the reference.